src/components/AtmosWorkflow/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import * as yaml from 'js-yaml';
+
+// Define constants for the base URL and workflows directory path
+const CLOUDPOSSE_DOCS_URL = 'https://raw.githubusercontent.com/cloudposse/docs/master/';
+const WORKFLOWS_DIRECTORY_PATH = 'examples/snippets/stacks/workflows/';
+
+async function GetAtmosTerraformCommands(workflow: string, fileName: string, stack?: string): Promise<string[] | undefined> {
+  try {
+    // Construct the full URL to the workflow YAML file
+    const url = `${CLOUDPOSSE_DOCS_URL}${WORKFLOWS_DIRECTORY_PATH}${fileName}.yaml`;
+
+    // Fetch the workflow file from the constructed URL
+    const response = await fetch(url);
+    if (!response.ok) {
+      console.error('Failed to fetch the file:', response.statusText);
+      console.error('Workflow URL:', url);
+      return undefined;
```

Comment:
Avoid doing a remote fetch. Instead do something like this with the raw loader: https://github.com/cloudposse/docs/blob/master/src/components/EmbedFile/index.js